06_proxmox_lvm_resize_disk_ssh_access_backups_25-aug-2022/lvm.md

```diff
@@ -1,24 +1,28 @@
-Демо на Disk Resize
+## Disk resize ##
+
+### Prerequisites: ###
+
+- VM with Rocky Linux
+- Root partition should be the last by order (though there is possibility to resize independent on partition order, this example does it for the simple case).
+- Discard checkbox should be checked (for fstrim usage).

```

Comment:
where is this checkbox? How it can be found? Please attach a screenshot (or even two screenshots, one for the case when VM is created and one when VM is already created and its parameters need to be updated)